### ISSUE TITLE
IMPLEMENTATION_STATUS should/could be generated automatically out of Sharness Tests

### ISSUE THREAD

**Issue by diasdavid** [ipfs/ipfs-sharness-tests#6]
https://github.com/ipfs/ipfs/blob/master/IMPLEMENTATION_STATUS.md

**Comment by chriscool** [comment]
Yeah there is already a kind of coverage script in the go-ipfs repo:

https://github.com/ipfs/go-ipfs/blob/master/test/sharness_test_coverage_helper.sh

It shows the existing tests for each ipfs command and each option of each IPFS command. For example, here is some output:

```
ipfs pin verify
      2 t0085

ipfs pin verify --quiet
ipfs pin verify -q

ipfs pin verify --verbose
      1 t0085

ipfs ping

ipfs ping --count
ipfs ping -n
```

It shows that `ipfs pin verify` is used twice in test t0085 while there are no tests for `ipfs ping` with or without the `--count` (`-n` is a shortcut for `--count`) option.

We could reuse that script to associate IPFS commands (and probably each option of each comamnd) with the tests they appear in and we could tell if they are working on not depending on whether the associated tests pass or not.
